app/assets/javascripts/Components/notes_table.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import {render} from 'react-dom';
+
+import ReactTable from 'react-table';
+import {admins_path, all_notes_notes_path, edit_note_path} from "../../../javascript/routes";
+
+class NotesTable extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      notes: [],
+      columns: {},
+      loading: true
+    };
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData = () => {
+    $.get({
+      url: Routes.all_notes_notes_path()
+    }).then(res => {
+      this.setState({
+        notes: res['notes_data'],
+        loading: false,
+        columns: res['column_headers']
+      });
+    });
+  };
+
+  buttons(modifiable, id) {
```

Comment:
Use `editable` instead of `modifiable`. Also, for methods that return JSX it's good to name them using "render". E.g., `renderButtons` instead of `buttons`. Applies throughout this file.